input/resources/measure-exm125-FHIR.json

```diff
@@ -712,9 +712,10 @@
           "display": "Initial Population"
         } ]
       },
+      "description": "Females between 51 and 74 years of age who have a qualifying outpatient visit.",
       "criteria": {
         "language": "text/cql-identifier",
-        "expression": "Initial Population"
+        "expression": "Patient.gender = 'female'\n\t\t\t\tand Global.\"CalendarAgeInYearsAt\"(FHIRHelpers.ToDate(Patient.birthDate), start of \"Measurement Period\") in Interval[51, 74]\n\t\t\t\tand exists AdultOutpatientEncounters.\"Qualifying Encounters\""
```

Comment:
These need to stay as references to the identifier of the expression, in-lining the expression like this is actually a violation of the conformance requirements.